src/tile/tile.scss

```
@@ -313,3 +315,30 @@
     @include visually-hidden;
   }
 }
+
+@mixin iui-tile-array {
+  width: 100%;
+
+  // #region IE support
+  .iui-tile {
+    margin: $iui-baseline $iui-sm;
+  }
+
+  @supports (grid-gap: $iui-line-height $iui-l) {
+    .iui-tile {
+      margin: 0;
+    }
+  }
```

Don't need to target `iui-tile`.

```suggestion
  > * {
    margin: $iui-baseline $iui-sm;
  }

  @supports (gap: 0 0) {
    > * {
      margin: 0;
    }
  }
```